### DISCUSSION TITLE
Add a "Reset Password" option

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5741] [Feature Requests]
<img width="759" alt="image" src="https://user-images.githubusercontent.com/37256050/218622869-46e537f2-c40b-4f5a-8395-e3bb6cb2ca8a.png">

Instead of setting a placeholder password and instructing a user to change it, there should be some option to reset the password where you can send them a single-use link so that they can set their own.

**Comment by maxpatiiuk** [comment]
How the change password form might look like:

<img width="1800" alt="Screenshot 2023-03-17 at 13 55 38" src="https://user-images.githubusercontent.com/40512816/225993778-aab72d01-d144-460e-ac9a-f0bd716435fb.png">

**Comment by bronwyncombs** [comment]
Requested from CSIRO Mineral Resources team via [asana](https://app.asana.com/0/1201972579963460/1209393684490373) and [discourse](https://discourse.specifysoftware.org/t/modify-login-page-reset-password/2243): 

They request this ability be added or have the ability to add it to the login page. 

> How can the user request a reset (forgot) password from the login window?
> ![image](https://github.com/user-attachments/assets/07b676a3-c5ab-4e9c-b03f-5f3c3916d35e)
>
